docs/architecture.md

```diff
@@ -647,6 +647,14 @@ If the data passed through the tool-parameters can trigger actions other than th
 * [tsx](https://github.com/privatenumber/tsx) is used to run the TypeScript code directly in Node.js during development, without a separate compilation step.
 * [ava](https://github.com/avajs/ava/) is used for testing
 
+## Distribution
+
+The UI5 MCP server is primarily published to the [official npm registry](https://www.npmjs.com/), making it easily accessible for Node.js-based development environments and MCP hosts that support npm package installation.
```

Comment:
I am not so happy with the term "primarily". This is the single point of truth location. @KlattG any ideas?

Reply:
ACK